src/metrics.ts

```diff
@@ -213,6 +213,11 @@ export function getMetrics(
       help: 'Number of times we include peers in a topic mesh for random reasons',
       labelNames: ['topic']
     }),
+    meshPeerInclusionEventsTotal: register.gauge<{ topic: TopicLabel }>({
```

Comment:
maybe we can rename these metrics. I think "inclusion/churn" is a bit more confusing than "add/remove"